_visualisation/core/parties-nodes.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Parties (Nodes)
+layout: visualisation
+---
+
+Nodes represent interested Parties, such as Persons, Entities or Regimes.
+{: .lead}
+
+
+## Appearance
+
+* Nodes are drawn as a circle containing an icon.
+  * Always draw circles/icons at the same size.
+  * Try to space nodes at regular intervals across the diagram.
+* The icon used indicates the type of Party that the node represents.
+  * See the [Vocabulary](/visualisation/core/vocabulary) for the valid types of Party and the correct icons to use for each.
+
+
+## Labels
+
+Nodes must have a Label underneath, identifying the Party. The Label can either be:
+
+* The Party's proper name (if known).
+* A unique Identifier that you create (see below).
+
+
+### Identifiers
+
+Instead of naming a Party in its Label, you can create an Identifier. Do this when identifying the name is not [relevant](/visualisation/core/relevance) or is not known. An Identifier is formed from:
+
+|      | Status +       | Type +             | Unique ID
```

Comment:
That an Identifier is a string formed from the concatenation of Status + Type + ID. Do you think that was already clear enough? Obviously the +'s have ended up confusing it rather than helping!

Reply:
Aha, yeah, I think it's the table spacing that's confusing things. Maybe you cold write it out like that first, then drop the + from the table headings?